src/components/tableV2/netdataTable.js

```diff
@@ -661,11 +662,10 @@ const renderActionWithDropdown = ({
       const actionRef = useRef()
 
       return (
-        <>
+        <React.Fragment key={id}>
```

Comment:
I think its safe to complete remove the Fragment here.

Reply:
@AlexNti There are 2 components inside - Action and ColumnsMenu, perhaps you missed it because of how GH collapses the code?

Reply:
Oh, yes you correct. I missed that